### DISCUSSION TITLE
Attributo Target file mkdocs.yml

### DISCUSSION THREAD

**Discussion by gbvitrano** [ondata/guidaPraticaPubblicazioneCSV#23] [Domande]
Inserendo nell'header la possibilità di cambiare lingua ho inserito nel file mkdocs.yml 
![image](https://user-images.githubusercontent.com/20488693/146824503-8fed1067-5fde-441e-b509-19acbff98436.png)
In questo modo la guida in italiano viene aperta in una nuova finestra, ma la vorrei fare aprire nella stessa finestra, per cui ho aggiunto l'attributo target, ma non funziona
![image](https://user-images.githubusercontent.com/20488693/146824902-b33d0fc7-dc28-4e76-8c29-aaca7cf65bdd.png)

C'è modo di far aprire i link delle varie lingue nella stessa finestra?

**Comment by aborruso** [comment]
Gianni, un mkdocs material multilingua vero, va gestito come sotto: tutto in un repo.
Nella guida ufficiale non è spiegato bene, qui sì https://www.javieranto.com/kb/GNU-Linux/Software/mkDocs/setup/changing-the-language/

![image](https://user-images.githubusercontent.com/30607/146828597-4d64bc0a-d051-4048-bc23-26671ede8449.png)

Nel sito attuale in ogni caso, tutti gli URL che puntano a un sito diverso hanno forzato il target a `_blank`.

Devo dirti, che per ora mi sto orientando per tenerli separati. Lasciamo la issue aperta e decidiamo a fine lavori.

**Comment by gbvitrano** [comment]
Grazie domani mi studio tutto :-)

**Reply by aborruso** [reply]
> Grazie domani mi studio tutto :-)

ribadisco però che probabilmente lì terremo distinti: perché quello spagnolo è un regalo per loro, quello italiano è un progetto che non si ferma alla traduzione, ma aggiunge cose.

**Comment by gbvitrano** [comment]
Ho fatto delle prove e mi sono incartato :-)  
Visto che comunque  devono essere distinte, la soluzione migliore e più semplice mi è sembrata quella che ho adottato, ovvero ho aggiunto una pagina `it.md ` con un refresh di 2 secondi  verso il link in spagnolo `<meta http-equiv="refresh" content="2; URL='https://ondata.github.io/guidaPraticaPubblicazioneCSV/'" />` e disattivando tutti contenuti della pagina.
Se la soluzione vi piace si dovrebbe inserire il menù multi lingua sul lato italiano.

![2021-12-21_17h26_41](https://user-images.githubusercontent.com/20488693/146964482-157e09dd-a07f-4333-a23e-b4c44028debc.png)
